neuralmonkey/trainers/regularizers.py

```diff
@@ -0,0 +1,139 @@
+"""Variable regularizers.
+
+This module contains classes that can be used as a variable regularizers
+during training. All implementation should be derived from the Regularizer
+class.
+
+"""
+from typing import List
+
+import numpy as np
+import tensorflow as tf
+from typeguard import check_argument_types
+
+from neuralmonkey.logging import log
+
+
+class Regularizer:
```

Comment:
podědili bych od ABCMeta. Podobně je to u attentionu myslim.

Reply:
BaseAttention dedi od modelpart

Reply:
ok, tak modelpart dědí od ABCMeta